app/assets/javascripts/ajax_ops.js

```diff
@@ -92,5 +108,147 @@ function handle_ajax(event) {
         }
       });
     }
+    // Delete a user
+    else if (event.target === deleteUserButton) {
+        fetch(`${users_path}/${delUserID.value}`,
```

Comment:
My rest application has validations on it, so I thought it will be like I am repeating my self: doing checks and validate data twice.
When I tested, I put some nil/empty values and it alert an error to introduce a valid data.